pkg/apis/app/v2/kieapp_types.go

```diff
@@ -152,18 +152,32 @@ type KieServerSet struct {
 	KafkaJbpmEventEmitters *KafkaJBPMEventEmittersObject `json:"kafkaJbpmEventEmitters,omitempty"`
 	Cors                   *CORSFiltersObject            `json:"cors,omitempty"`
 	// MDBMaxSession number of KIE Executor sessions
-	MDBMaxSession *int `json:"MDBMaxSession,omitempty"`
+	MDBMaxSession    *int              `json:"MDBMaxSession,omitempty"`
+	TerminationRoute TerminationObject `json:"termination,omitempty"`
+}
+
+type TerminationObject struct {
+	// If set to true disableSSL will be skipped for the console the target port will be http and the termination will be edge
+	EnableInsecureEdgeTermination bool `json:"enableInsecureEdgeTermination,omitempty"`
```

Comment:
Not sure if it is a good idea to ignore disableSsl property. We should be probably checking bot properties. Also it would be good to provide better description for this property, for someone it maybe look confusing the part with `set to true disableSSL`. I think that description starting like "If this property is set to true then ..,."

Reply:
Is not disabled, is just overrided for the specific obejct, disableSSL is global and if someone want to use edge termination and http only on console we are going to enabled this behaviour